unikernel.ml

```diff
@@ -1810,41 +1859,31 @@ struct
         | "/unikernel/deploy" ->
             check_meth `GET (fun () ->
                 authenticate store reqd (deploy_form store reqd))
-        | "/unikernel/destroy" ->
+        | "api/unikernel/destroy" ->
```

Comment:
why is there suddenly no leading `/`?

Reply:
this should be an error. I'll fix that in a commit